docs-gb/kubernetes/resources/pipeline.md

```diff
@@ -16,9 +33,93 @@ The output of the Pipeline is the output from the `tfsimple3` model.
 
 The full GoLang specification for a Pipeline is shown below:
 
-```{literalinclude} ../../../../../../operator/apis/mlops/v1alpha1/pipeline_types.go
-:language: golang
-:start-after: // PipelineSpec
-:end-before: // PipelineStatus
+```go
```

Comment:
Please reference the file via a @github-files/github-code-block. As much as possible, we want to change things in a single place, with the files in the repo being the source-of truth.

Reply:
Hi @lc525 please help me with the location of file. Unable to find here: https://github.com/SeldonIO/seldon-core/tree/v2/k8s/samples

Reply:
fixed